docs/source/workshop.rst

```diff
@@ -427,4 +426,44 @@ Users with appropriate access can comment on a submission and reply to other's c
     :align: center
 
 
+Extracting all notes given an invitation
+-------------------------------------------
+
+Users can access all notes by using an invitation id. 
+Important note: only the notes readable by the account the user logged in with can be accessed.
+Consider the following example which gets all the papers submitted to ICLR 2018::
+
+	>>>iclr18_submissions = client.get_notes(invitation="ICLR.cc/2018/Conference/-/Submission")
+
+Please note, get_notes() will return only the first 1000 notes. If you expect the number of notes returned to be more, you will require tools.iterget_notes() which returns an iterator and lets you access all the notes.
+
+
+Extracting all Public Comments for a conference
+--------------------------------------------------
+
+Comments, just like submissions, are saved as notes. So they are also accessible using get_notes().
+Note that the invitation argument in get_notes() accepts regex.
+Consider the following example to get all the "Public Comments" made on submissions in ICLR 2019 conference.::
+
+	>>>iclr19_public_comments = client.get_notes(invitation="ICLR.cc/2019/Conference/-/Paper.*/Public_Comment")
+
+Extracting comments made on a forum
+----------------------------------------
+
+All comments made on a particular forum/submission can be extracted like this::
+
+	>>>iclr19_forum_comments = client.get_notes(forum="<forum-id>")
+
+Also, the public comments on a particular forum can be extracted like this:: 
```

Comment:
You may want to point out that "Public_Comment" is an arbitrary string - it can be anything. It just so happens that we use this naming convention in ICLR.

Reply:
I'm not 100% sure about this advice, though. It's true that it's an arbitrary string, but maybe we should be treating it like a convention, and encouraging (forcing?) other conferences to use it.

Reply:
@mspector I just added something to this end. Let me know if I should write more or maybe in a different way to convey this idea.